pkgs/top-level/static.nix

```diff
@@ -243,4 +243,6 @@ in {
   ) super.ocaml-ng;
   
   python27 = super.python27.override { static = true; };
+
+  oniguruma = super.oniguruma.overrideAttrs (_: { cmakeFlags = ["-DBUILD_SHARED_LIBS=OFF"]; });
```

Comment:
I think it is more standard for these sort of per-package overrides to be documented in the package  definition itself. That said, I'm not certain why these other ones are in here :)

Reply:
Oops, I thought this was the convention. What's a good model for me to refer to?

Reply:
Typically you add an explicit `static` (or `staticOnly`) parameter to the package expression. Take a look at brotli, you should be able to copy the flags handling verbatim.